presto-main/src/main/java/com/facebook/presto/execution/AddColumnTask.java

```diff
@@ -36,10 +36,12 @@
 
```

Comment:
You can improve a commit message a bit:

```
Add support for NOT NULL in DDL statements

Enable NOT NULL in the parser and add a flag in the
metadata to check whether or not a connector supports this new
functionality.  If a connector doesn't support NOT NULL semantics in
columns, yet this is supplied, then a semantic exception is thrown
indicating to the user that this functionality is not supported in their
catalog.
```

Reply:
Extrapolating my findings from this feedback.  I'll be changing:

`This change enables NOT NULL in the parser and adds a flag in the metadata [...]`

to

`Enable NOT NULL in the parser and add a flag in the metadata [...]`

Because I did not use the imperative mood, as per: https://chris.beams.io/posts/git-commit/